### PR TITLE
update introduction on majifix components

### PR DESCRIPTION
**Pull request by joshuamabina**
Expand components descriptions to be paragraphs in-stead of being just a single or two lines.

### CODE REVIEW

**Comment by joshuamabina** [comment]
Like discussions we had at #2, the pull-request updates the introduction part so that the majifix components descriptions are further expanded.

TODO: 
- Please provide available repository urls for majifix components.

File: get-started.md [hunk 1]
```diff
@@ -16,14 +16,49 @@ support public service providers in solving each and every problem, while giving
...
+
+You can consider it as the data layer within a platform (including the data model). It is designed to be plugged into a templating layer, a mobile application or to be used with any other data consumer.
+
+Majifix API provides a starting point that's further advanced than a framework. It allows you to get a complete data layer up and running in minutes.
+
+See source code [repository](https://github.com/CodeTanzania/open311-api).
+
+### Majifix management app
+
+Majifix management app is desined to help manaes statistical information about system performance to official city managers.
```
Comment (BenMaruchu) [inline get-started.md]: There is a typo on this line.

File: get-started.md [hunk 2]
```diff
@@ -16,14 +16,49 @@ support public service providers in solving each and every problem, while giving
...
+
+See source code [repository](https://github.com/CodeTanzania/open311-mobile-manager).
+
+### Majifix client mobile app
+
+Majifix mobile app is used by citizens to help them instantly report problems to city managers
+
+See source code [repository]().
```
Comment (BenMaruchu) [inline get-started.md]: This is still a private repo so the source code will be published once it is refactored and all sensitive information removed from the code

File: get-started.md [hunk 3]
```diff
@@ -16,14 +16,49 @@ support public service providers in solving each and every problem, while giving
...
+
+### Majifix facebook messenger bot
+
+Majifix facebook messenger bot is designed to create more social engagement by allowinj citizens to submit problems to the Majifix platform through Facebook Messenger's interface without downloadin the actual app.
```
Comment (BenMaruchu) [inline get-started.md]: This a typo also on this line

**Comment by BenMaruchu** [comment]
Hey @joshuamabina  thanks for the PR, I have added my comments on the PR please review them.

**Review by BenMaruchu — COMMENTED** [review]
Here are some of my comments regarding this PR

**Pull request merged**